app/views/shelters/show.html.erb

```diff
@@ -1,17 +1,32 @@
 <h2><%= @shelter.name.upcase %></h2>
 
 <div>
-  <seciton id='shelter-info'>
+  <section id='shelter-info'>
     <%= @shelter.address %>
     <%= @shelter.city %>
     <%= @shelter.state %>
     <%= @shelter.zip %></br></br>
   </section>
 </div>
 
+<div>
+  <section id='shelter-reviews'>
```

Comment:
For the section ID, we will most likely want an "individualized" section for each review.  This will allow us to use the "within" testing feature we got a glimpse of today in class.

Reply:
Reviewed and updated code together to use individualized sections